service/service.go

```diff
@@ -51,6 +51,11 @@ func DefaultConfig() Config {
 	}
 }
 
+func (c Config) GoString() string {
+	return fmt.Sprintf("service.Config{Description:%q, GiCommit:%q, Name:%q, Source:%q, Viper:%v}",
+		c.Description, c.GitCommit, c.Name, c.Source, c.Viper.AllSettings())
+}
```

Comment:
What is this good for?

Reply:
I realized logging Viper contents may expose some sensitive information like tenantID. I don't think it's desired. I replaced logging config with just logging gitCommit. I don't find other information (besides Viper state) really useful.